database/src/main/resources/changesets/backend_KV_Store.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+    <changeSet author="rmunshi" id="BACKEND_KV_STORE">
+        <comment>
+            Storage for all Backend K/V Pairs.
+        </comment>
+        <createTable tableName="BACKEND_KV_STORE">
+            <column autoIncrement="true" name="BACKEND_KV_STORE_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_BACKEND_KV_STORE_ID"/>
+            </column>
+            <column name="WORKFLOW_EXECUTION_UUID" type="VARCHAR(255)">
```

Comment:
TOL I called this `WORKFLOW_UUID` in the Job Store. Not sure which is more accurate but on reflection, yours is probably better